Can't start simple example (examples/simple/main.go)

Running ``main.go`` (with deps installed) gives me the following:

```
bot|⇒ go run main.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x124521]

goroutine 1 [running]:
panic(0x384aa0, 0xc82000a0c0)
	/usr/local/Cellar/go/1.6.2/libexec/src/runtime/panic.go:481 +0x3e6
github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus.(*Entry).Reader(0xc820012700, 0xc83448c104, 0x0, 0x0)
	/Users/onbjerg/Go/src/github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus/entry.go:44 +0x61
github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus.Entry.log(0xc820012580, 0xc820016fc0, 0x0, 0x0, 0x0, 0xc82000b100, 0x0, 0x0, 0x4, 0xc82000b180, ...)
	/Users/onbjerg/Go/src/github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus/entry.go:94 +0x296
github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus.(*Entry).Info(0xc820051ce8, 0xc820051e10, 0x1, 0x1)
	/Users/onbjerg/Go/src/github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus/entry.go:129 +0x8c
github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus.(*Logger).Info(0xc820012580, 0xc820051e10, 0x1, 0x1)
	/Users/onbjerg/Go/src/github.com/broadway33/bot/vendor/github.com/Sirupsen/logrus/logger.go:128 +0xe7
github.com/broadway33/bot/vendor/github.com/danryan/hal.(*Robot).Run(0xc820054070, 0x0, 0x0)
	/Users/onbjerg/Go/src/github.com/broadway33/bot/vendor/github.com/danryan/hal/robot.go:94 +0x126
main.run(0xc820000180)
	/Users/onbjerg/Go/src/github.com/broadway33/bot/main.go:31 +0x21f
main.main()
	/Users/onbjerg/Go/src/github.com/broadway33/bot/main.go:40 +0x18
exit status 2
godep: go exit status 1
```

Seems like the error starts when the logger is used in robot.go at line 94. Can confirm that using the logger anywhere will give a similar error (``hal.Logger``).

--- a/config.go
+++ b/config.go
@@ -30,9 +30,10 @@ func newLogger() *logrus.Logger {
 	if err != nil {
 		panic(err)
 	}
-	return &logrus.Logger{
-		Level: level,
-	}
+	logger := logrus.New()
+	logger.Level = level
+	
+	return logger
 }
 
 // newRouter initializes a new http.ServeMux and sets up several default routes

